src/main/resources/META-INF/JS/CheckInCheckOutAjax.js

```diff
@@ -0,0 +1,151 @@
+            }
+        },
+            errorReason => {
+            console.log(`An error occured in the fetch api\nreason: ${errorReason}`);
+            printError(errorReason);
+            })
+        .then(data => {
+            console.log(data)
+        });
+}
+
+async function getCheckoutBookForStudentAndTermAJAX(loginUserInfo, studentInfo, studentBookInfo){
+
+    getAllCheckedOutBooksForStudentAndTermAJAX(loginUserInfo, studentInfo);
+
+    fetch(studentBookInfo, {
+            method: "GET",
+             headers: {
+                'Content-Type': 'application/json',
+                'Accept': 'application/json'
+            },
+            body: ({
+                BarCode: "barcode",
+                StudentInfo: "studentInfo"
+
+            })
+        })
+            .then(response => {
+                if(response.ok){
+                    return response.json();
+                }
+                else{
+                    console.log("An Error Occurred")
+                    throw Error("Error");
+                }
+            },
+                errorReason => {
+                console.log(`An error occured in the fetch api\nreason: ${errorReason}`);
+                printError(errorReason);
+                })
+            .then(data => {
+                console.log(data)
+            });
+}
+
+async function sellBookForStudentAJAX(loginUserInfo, studentBookInfo){
+    fetch(loginUserInfo, {
+        method: "GET",
+        headers: {
+            'Content-Type': 'application/json',
+            'Accept': 'application/json'
+        },
+        body: ({
+            username: "_username",
+            passowrd: "_password"
+        })
+    })
+        .then(response => {
+            if(response.ok){
+                return response.json();
+            }
+            else{
+                console.log("An Error Occurred")
+                throw Error("Error");
+            }
+        },
+            errorReason => {
+            console.log(`An error occured in the fetch api\nreason: ${errorReason}`);
+            printError(errorReason);
+            })
+        .then(data => {
+            console.log(data)
+        });
+
+    fetch(studentBookInfo, {
+        method: "GET",
+        headers: {
+            'Content-Type': 'application/json',
+            'Accept': 'application/json'
+        },
+        body: ({
+            BarCode: "barcode",
+            StudentInfo: "studentInfo"
+
+        })
+    })
+        .then(response => {
+            if(response.ok){
+                return response.json();
+            }
+            else{
+                console.log("An Error Occurred")
+                throw Error("Error");
+            }
+        },
+            errorReason => {
+            console.log(`An error occured in the fetch api\nreason: ${errorReason}`);
+            printError(errorReason);
+            })
+        .then(data => {
+            console.log(data)
+        });
+}
```

Comment:
I don't know how JavaDoc works in  JavaScript but i don't see any documentation or comments in this class